CHANGELOG.md

```diff
@@ -48,9 +53,6 @@ v0.36.0 (2023-08-30)
 - `otelcol.exporter.prometheus`: Set `include_scope_info` to `false` by default. You can set
   it to `true` to preserve previous behavior. (@gouthamve)
 
-- `prometheus.remote_write`: Set `retry_on_http_429` to `true` by default in the `queue_config` block.
-  You can set it to `false` to preserve previous behavior. (@wildum)
-
```

Comment:
But in flow mode this change did make it to 0.36.0: https://github.com/grafana/agent/blob/release-v0.36/component/prometheus/remotewrite/types.go#L33

Also, if we wanted to change something in the 0.36.0 heading, we would need to also fix it in the `release-v0.36` branch.